index.bs

```diff
@@ -1719,6 +1719,45 @@ that should be considered in the development of new features, notably:
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3>Extend existing manifest files rather than creating new ones</h3>
+
+Manifest files are sometimes required. However, they can also add complexity to the web platform,
+which creates developer confusion. The use of different manifests on the web should therefore be kept to a minimum.
+
+Metadata can reside inside a new manifest if that is related to a different function or the fetch timing 
+of that metadata is different. Application metadata should be added to the Web App Manifest or be an extension of it. 
+Parameters for a specific function (like Payment Handler Manifest) are a completely different type. Publication Manifest,
+using JSON-LD is another type as well.
+
+Note that in all cases, the naming conventions should be harmonised (see section 2.2)
+ 
+[Note: Due to historical reasons, existing manifests use lowercase, underscore-delimited names. There have been times 
+where it has been beneficial to re-use dictionaries from a manifest in DOM APIs as well, which are camel-cased. 
+One such example is the <a href="https://w3c.github.io/image-resource/">image resource</a>. For this reason, if a 
+key can clearly be expressed as a single word, that is recommended.]
+ 
+When designing new keys and values for a manifest, make sure they are really needed (thus, define well thought out use-cases).
+Also please check if a similar key exists. If an existing key/value pair does more or less what is needed, 
+work with the existing spec to extend it to your use-case if possible.
+
+Some of the existing manifest files include
+
+* <a href="https://www.w3.org/TR/appmanifest/">Web App Manifest</a>
+* <a href="https://www.w3.org/TR/payment-method-manifest/">Payment Method Manifest</a>
+* <a href="https://www.w3.org/TR/pub-manifest/">Publication Manifest</a>
+
+We encourage people to think of manifest files as extensible. So attempt to get the changes into the original spec, 
+or at least have a discussion around the extension with the spec editors, as that will usually lead to a better design
+and to something that can be integrated into the spec. There are certain times the spec authors might not want to 
+integrate immediately due to process (like going to CR) or due to it having a different scope, like extension to Web App
+Manifest only affecting store or payment use-cases. In that case it is fine to monkey-patch as long as that is agreed 
+with the original spec editors.
+
+An example of something that was done as a monkey patch but scheduled to be integrated into the web app manifest in 
+a future level (post CR):
+
+* https://wicg.github.io/web-share-target/#extension-to-the-web-app-manifest 
```

Comment:
Kind risky linking to this because it will break once we move it over... What would be idea is, "new property is defined in existing manifest, but processing of the new property can be defined in another spec".

Reply:
Since process wise it has already moved out of incubation, could you give us the new URL? (if it exists, that is.)

Otherwise you could just send a PR whenever the new repo is ready.

Reply:
Still in incubation... but yes, can try to remember to send a PR.

Reply:
Okay, then remind us later.